Protos/V1/applications_common.proto

```diff
@@ -17,11 +18,26 @@ option csharp_namespace = "ArmoniK.Api.gRPC.V1.Applications";
 message ApplicationRaw {
   string name = 1; /** Application name. */
   string version = 2; /** Application version. */
-
   string namespace = 3; /** Application namespace used in the excecuted class. */
   string service = 4; /** Application service used in the excecuted class. */
 }
 
+/**
+ * Represents every available field in an application.
+ */
+enum ApplicationRawField {
+  APPLICATION_RAW_FIELD_NAME = 0; /** Application name. */
```

Comment:
Do we put an unspecified in each for coherence sake ?

Reply:
Yes, on peut faire ça. J'ajoute